ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -374,6 +374,15 @@ static void* dasm_labels[zend_lb_MAX];
 ||	}
 |.endmacro
 
+/* The caller is responsible for passing the parameters, if there are any.
+ * If there are any parameters, the way that is done depends on whether the function is declared as ZEND_FASTCALL.
+ *
+ * With ZEND_FASTCALL: This file's usage of 'EXT_CALL zend_is_true'
+ * is an example of a single-parameter function getting called.
+ *
+ * Without ZEND_FASTCALL: This file's usage of 'EXT_CALL zend_interrupt_function'
+ * is an example of a single-parameter function that is not ZEND_FASTCALL getting called.
+ */
```

Comment:
``EXT_CALL func`` is a macro that translated to ``call func`` if possible (offset between current %IP and *func* fits into signed 32-bit) or ``movabs func, %r; call %r`` (if it's too big). It's not related to FASTCALL calling convention. Remove this comments.

Reply:
Done